STEERING-COMMITTEE.md

```diff
@@ -100,19 +97,25 @@ first name):
 \* Open seats have been allocated to the organization per the committee rules,
   but have not been assigned to a member of the organization.
 
-### Allocation of seats
+### Composition
+
+As of 2020, the three organizations with the largest demonstrated 
+commitment to the project are Google, IBM/RedHat, and VMWare. For
+2021, a 5 person steering committee should be competent to
+execute its responsibilities, with three seats for Google, one seat
+for IBM/RedHat, and one seat for VMWare.
 
-Seats on the steering committee are allocated based upon contribution to the
-project by an organization. No final decision has been made on the exact
-formula.
+### Nomination of representatives
 
-As the project continues to grow, we expect to add additional seats to the
-committee, ensuring that those contributing to the project are properly
-represented.
+Seats on the steering committee are allocated based upon lifetime
+contributions to the project by an organization. The intention is
+to ensure that those organizations are committed to the ongoing
+maintenance burden that adding to core implies, and express their
+commitment to the project and the project's users.
 
 - After a seat is allocated to an organization, the organization shall nominate
   a candidate to be confirmed by KSC. The committee reserves the right to not
-  confirm a candidate, in which the organization would need to nominate a new
+  confirm a candidate, in which case the organization would need to nominate a new
```

Comment:
This feels a bit more slippery as a long-term solution with a structural vendor majority (observation).

Overall, this seems like a reasonable rule to prevent truly obnoxious behavior, but it's like bleach and ammonia -- two reasonable cleaners that are toxic when mixed.